docker-compose.yml

```diff
@@ -14,29 +17,6 @@ services:
       - SNOWFLAKE_DB_SCHEMA
       - SNOWFLAKE_DB_WAREHOUSE
   dev:
-    image: keboola/ex-db-snowflake
-    command: composer ci
+    <<: *base
```

Comment:
woah, this is some ancient Egyptian hack? :-)  I will use this going forward 👍

Reply:
https://viewsourcecode.org/why/redhanded/bits/yamlSMergeKey.html